ZenPacks/zenoss/Microsoft/Windows/WinService.py

```diff
@@ -109,6 +109,74 @@ def monitored(self):
                     if self.getMonitored(datasource):
                         return True
 
+        # 3 check the service class
+        if hasattr(self, 'serviceclass') and 'serviceclass' in self.getRelationshipNames():
```

Comment:
That's fine, it is more that funny dance of checking for the existance of the attribute and relationship that is confusing to me.